cypress.json

```diff
@@ -1,9 +1,9 @@
-{
+{ 
   "baseUrl": "https://folio-snapshot.dev.folio.org",
   "viewportWidth": 1920,
   "viewportHeight": 1080,
   "video": false,
-  "defaultCommandTimeout": 31000,
+  "defaultCommandTimeout": 21000,
```

Comment:
what for this change?

Reply:
I don't know, but I'll fix it for 31000)

Reply:
I don't know, but I'll fix it for 31000)